CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.2.2
```

Comment:
How do you normally choose when to bump the feature version? Support for a new language feature could justify it?

Reply:
I'm usually pretty conservative. This isn't a breaking change and doesn't really add any new "features" (in terms of the CLI arguments or library API). Also, it doesn't change the formatting of any existing code. That sounds like a point release to me, even though it's significant in that it adds support for new syntax. But since no code using that syntax exists in the wild, from the user's perspective, there's no visible change in dart_style at all.